src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,87 @@
+import model from '../db/models';
+
+const { Bookmark } = model;
+/**
+ * Bookmark class
+ */
+class BookmarkManager {
+  /**
+     *
+     * @param {Object} req
+     * @param {Object} res
+     * @return {Object} bookmarked story
+     */
+  static async create(req, res) {
+    try {
+      const { id } = req.user;
+      const { slug } = req.params;
+
+      const createBookmark = await Bookmark.create({
+        userId: id,
+        slug
+      });
+      return res.status(201).json({
+        message: 'You have bookmarked successfully',
+        data: createBookmark
+      });
+    } catch (error) {
+      return res.status(500).json({
+        error: 'You have already bookmarked this story'
+      });
+    }
+  }
+
+  /**
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} Bookmarked stories
+   */
+  static async getAllUsersBookmark(req, res) {
```

Comment:
Try to change the methods name. What I think is that the bookmark was for articles not for users...